### DISCUSSION TITLE
New error I hadn't previously : StoryTellerException: Could not locate device partition

### DISCUSSION THREAD

**Discussion by elar1** [marian-m12l/studio#349] [Q&A]
Hello,

I didn't use the STUdio for a while, and maybe I shouldnt open the luniistore first, because now, it is not working anymore :'(
I have the last version of the STUdio.
Here is the error : 
2023-02-26 20:54:10,722 [ERROR] [s.d.LibUsbDetectionHelper]: An error occurred while handling device plug event
java.util.concurrent.CompletionException: studio.driver.StoryTellerException: Could not locate device partition

Do you have any idea how to fix this issue? Thanks a lot!

**Comment by blaiseli** [comment]
This looks like the same error message as [here](https://github.com/marian-m12l/studio/discussions/464) and [here](https://github.com/marian-m12l/studio/discussions/335)
